v2/session/common-customizations/sessions/multiple-api-endpoints.mdx

```diff
@@ -28,9 +28,9 @@ To enable use of sessions for multiple API endpoints, you need to use the `sessi
 - `sessionTokenBackendDomain` in the frontend config must match the `cookieDomain` set in the backend config.
 :::
 
-## Step 1) Backend config
+## Step 1) Set Cookie Domain in the Backend Config
 
-You need to set the `sessionTokenBackendDomain` value to be the common top level domain. For example, if your API endpoints are `{"api.example.com", "api2.example.com", "api3.example.com"}`, the common portion of these endpoints is `".example.com"` (The dot is important). So you would need to set the following:
+You need to set the `cookieDomain` value to be the common top level domain. For example, if your API endpoints are `{"api.example.com", "api2.example.com", "api3.example.com"}`, the common portion of these endpoints is `".example.com"` (The dot is important). So you would need to set the following:
```

Comment:
```suggestion
You need to set the `cookieDomain` value to be the common top level domain. For example, if your API endpoints are `{"api.example.com", "api2.example.com", "api3.example.com"}`, the common portion of these endpoints is `".example.com"` (The dot is NOT important). So you would need to set the following:
```

Reply:
The code snippets below must not have a `.` in the value set for cookie domain

Reply:
And also the text `The above will set the session cookies' domain to .example.com, allowing them to be sent to *.example.com.` will have to change (this is below the code snippet)

Reply:
The note that is below in this section will also need to change since it says "whilst the cookieDomain can start with a leading `.` ...), but in the snippet we won't be writing it with a leading dot.

Reply:
done